R/fct_SQLite.R

```diff
@@ -372,7 +356,9 @@ db_save_review <- function(
     "FROM",
     "row_updates",
     "WHERE",
-    sprintf("%s.id = row_updates.id", tables)
+    sprintf("%s.id = row_updates.id", tables),
+    "AND",
+    sprintf("%s.reviewed <> row_updates.reviewed", tables)
```

Comment:
Maintains current structure where updates only occur when reviewed status is being changed/updated.

Reply:
Note that the `tables` argument could contain multiple names to update in the database, but not anymore since your last changes. Maybe change it to 'table' and let it now error when multiple tables are entered (and update the description).